api/app/signals/apps/sigmax/stuf_protocol/outgoing/stuf.py

```diff
@@ -45,14 +45,18 @@ def _send_stuf_message(stuf_msg: str, soap_action: str):
         'Content-Length': b'%d' % len(encoded)
     }
 
+    cert = None
+    if settings.SIGMAX_CLIENT_CERT and settings.SIGMAX_CLIENT_KEY:
+        cert = (settings.SIGMAX_CLIENT_CERT, settings.SIGMAX_CLIENT_KEY)
+
     # Send our message to Sigmax. Network problems, and HTTP status codes
     # are all raised as errors.
     try:
         response = requests.post(
             url=settings.SIGMAX_SERVER,
+            cert=cert,
             headers=headers,
-            data=encoded,
-            verify=False
+            data=encoded
```

Comment:
Why is `verify=False` removed? The default value is `True`. Could you also make this configurable? So when `cert=None` the `verify=False` and when there is a cert `verify=True`?

Reply:
@vanbuiten I think from a security perspective it is wrong to set verify to False. I don't think we require verify to be False for our deployments any more, that is why I removed it. Is there a reason to make it configurable?

Reply:
@bartjkdp only reason to keep it is that if no cert has been configured the functionality is not changed, everything keeps working as is. I am also not sure if this will give any issues at Amsterdam 🤔

Reply:
@vanbuiten I don't think that the proposed change will give any issues at Amsterdam. When we moved to Azure we also changed the connection string to Sigmax from IP to hostname. So the SSL certificate presented by the Sigmax System now matches the hostname.